.github/workflows/upgrade-liff-sdk.yml

```diff
@@ -0,0 +1,31 @@
+name: Upgrade LIFF SDK
+
+on:
+  schedule:
+    - cron: '0 10 * * 1'
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
```

Comment:
It's better to use the LTS version. And, could you add `.nvmrc` or `.node-version` and then change this version with the same as it?